proto/user.proto

```diff
@@ -20,15 +20,14 @@ message AuthInfo {
 
 /// User defines a user in the system.
 message UserInfo {
-  uint32 id = 1;
-  string name = 2;
-  bool is_supper = 3;
-  bool can_create_db = 4;
-  bool can_login = 5;
-  AuthInfo auth_info = 6;
+  string name = 1;
```

Comment:
Perhaps user.id is useful in some scenarios? Not very sure, but we may add it back if necessray